content/en/blog/falco-0-38-0/index.md

```diff
@@ -0,0 +1,153 @@
+
+### Choose which rules to load at runtime 📝
+
+We distribute several files that contain community contributed rules and you can always write your own. But how do you select which rules Falco will load at runtime? There are several ways, including using `override`s or specifying command line options such as `-D`, `-t` and `-T`. However, those do not allow you to express something as simple as "I would like to exclude all rules except for this one" or "I would like to include a specific tag and disable some of its rules". Furthermore, you couldn't specify this configuration in your `falco.yaml` file. To make this possible, we introduced a new configuration option, `rules`, that can be specified both in the configuration file or the command line. For instance, you can now write:
+
+```yaml
+rules:
+  - disable:
+      rule: "*"
+  - enable:
+      rule: Netcat Remote Code Execution in Container
+  - enable:
+      rule: Delete or rename shell history
+```
+
+To finely control your rule loading without modifying the rule files themselves. Read more in [controlling rules](/docs/rules/controlling-rules/#via-falco-configuration-or-parameters).
+
+### Field transformers and value comparison in conditions 🟰
+
+Up until now we couldn't write a condition that catches operations like "a process deleting its own executable" because you couldn't use a field value on the right hand side of the condition. Since this version we have added a syntax to do just that with the `val()` operator:
+
+```
+evt.type = unlink and proc.exepath = val(fs.path.name)
+```
+
+will trigger only if the process exepath is the same as the unlink argument target, meaning that the process is trying to delete its own executable!
+
+In addition you can also apply simple transform operators to both sides of the comparison: `toupper()` and `tolower()` will convert casing as you'd expect and `b64()` can decode base64. Stay tuned for additional transformers to cover more use cases! Read more on [transform operators](/docs/rules/conditions/#transform-operators) in the documentation.
+
+### Prometheus Metrics support 🔥
+
+If you have been following Falco development, you probably know we are constantly improving support for metrics that tell you how the Falco engine is doing. We now have introduced [Prometheus support](/docs/metrics/falco-metrics/#prometheus-support) so you can better integrate Falco with your existing performance monitoring infrastructure.
+
```

Comment:
I'd add a note about an official grafana dashboard that will, sooner or later, be integrated in our charts.

Reply:
Added a note for it.